recipes/tdameritrade/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "tdameritrade" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 692baf738dc8c5f9250764ac7db101171c5df8dd59e39bae77d2a7ff41a02479
+
+build:
```

Comment:
It looks like this has an entry point that should be included:

https://github.com/timkpaine/tdameritrade/blob/master/setup.py#L64
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

Also add a `commands` test:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands